client/src/transmitter/index.ts

```diff
@@ -190,20 +193,22 @@ function start_recording(stream: MediaStream) {
   const connectTime = Date.now()
   connect(
     (openEvent) => {
-      setup_media_recorder_listener()
+      start_media_recorder()
     },
     (closeEvent) => {
-      // if (!recording) {
-      //   return
-      // }
-      // stop_recording()
-      // const connectionAge = Date.now() - connectTime
-      // const shouldRetry =
-      //   closeEvent.code === 1006 && connectionAge >= minRetryThreshold
-      // if (shouldRetry) {
-      //   console.log('restarting streaming due to ws 1006 error')
-      //   start_recording(stream)
-      // }
+      if (!recording) {
+        return
+      }
+
+      stop_recording()
```

Comment:
When I tested uncommeting this code locally, having this here basically stopped the recording after switching the media being displayed. It went like: 
 - change display button clicked
 - recording stopped by refresh_recording which closes the websocket
 - recording restarted by refresh_recording
 - this onClose function being called and stopping the recording exact at this point here
 - non-profit

aint you getting that on your console? the recording just stops after switching 😬

Reply:
I am preventing it from `stop_recording` if the error code is 1000.